README.md

```diff
@@ -38,6 +38,12 @@ Follow these steps to use the migration Lambda function:
 
 If you wish to use [AWS CLI](https://docs.aws.amazon.com/cli/latest/)
 This reduces the need to navigate around AWS Console which is always in flux and not the easiest to figure out.
+Ensure that in cognito > new user pool > App Clients > Client being used for login
+that Security configuration > Prevent User Existence Errors is set to legacy recommended https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-managing-errors.html
+
+This kicks in the lambda workflow that allows the migration lambda function to execute when a login failure occurs.
+Without this, the lambda function will not get called and will not execute.
+
```

Comment:
The English in this whole section is hard to understand :/

Reply:
> Do you have a reference why the changes to the skeleton are needed? It looks like you're just moving around a parameter -- not bad, and probably easier to work with, but would like to have a bit of rationale here.

Yes this one took me a while, the zipfile in the skeleton apparently requires the contents of the zipfile not a uri
https://itecnote.com/tecnote/amazon-web-services-aws-cli-create-lambda-function-cannot-unzip-uploaded-file/

It's easier and more rational to pass the zip file location as a parameter 

> The English in this whole section is hard to understand :/

Fair - how does this sound?

Cognito requires the user pool client Security Configuration > Prevent User Existence Errors is set to Legacy for migration lambdas to executed when a login failure occurs.
This can be accomplished with 

```
aws cognito-idp update-user-pool-client
--user-pool-id <value>
--client-id <value>
--prevent-user-existence-errors LEGACY
```

Reply:
> Yes this one took me a while, the zipfile in the skeleton apparently requires the contents of the zipfile not a uri

I think this might a difference in which CLI you use, I notice a `fileb` in there. Previous versions (potentially also on different OSs) may have pulled that in properly. [...]

> It's easier and more rational to pass the zip file location as a parameter

[...] Absolutely agree.